packages/test/end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -218,4 +233,12 @@ describe("Loader.request", () => {
         assert.strictEqual(sameDataStore1, sameDataStore2,
             "same containers do not return same data store for same request");
     });
+    it("request can handle url with query params", async () => {
```

Comment:
nit: Consider changing the description to something like "can handle url with query params". This will make the whole description including the `it` keyword as  as "it can handle url ..."